config/initializers/cycle_timetables.rb

```diff
@@ -37,12 +37,29 @@
     show_deadline_banner: Time.zone.local(2025, 7, 1, 9), # 12 weeks before Apply deadline
     apply_deadline: Time.zone.local(2025, 9, 16, 18),
     reject_by_default: Time.zone.local(2025, 9, 24, 23, 59, 59), # 1 week and a day after apply deadline
-    find_closes: Time.zone.local(2025, 10, 1, 23, 59, 59), # The evening before the find opens in the new cycle
+    find_closes: Time.zone.local(2025, 9, 30, 23, 59, 59), # The evening before the find opens in the new cycle
```

Comment:
Is there a card to change this too? https://github.com/DFE-Digital/publish-teacher-training/blob/main/app/services/find/cycle_timetable.rb#L43

What happens between the 24th of September and the 30th, i.e are courses still shown on Find but people are unable to apply for them?

Reply:
They are not able to apply for anything until 'apply_opens' in the next cycle. It's like this:
- Apply closes: Candidates can still view the courses they applied for on Find. If they roll over their application, they cannot select courses
- Find closes: They cannot view the courses they applied for on Find.
- Find reopens the next day: Now that can view courses for the next cycle. If they have rolled over their applications, they can select courses, but cannot apply
- Apply opens: BAU, they can select their courses and submit their applications

This is nothing new -- it's how the 2024 cycle is also working. There was discussion on slack about whether find should close when the apply deadline passes, but agreed that it would be unhelpful to anyone still making a decision about what offer to accept.

Reply:
Will do a quick PR now to change the date in publish. Good shout

Reply:
https://github.com/DFE-Digital/publish-teacher-training/pull/4443 for updating the dates in publish

Reply:
Ah yes that has jogged my memory now. Thank you for the very clear explanation!

So when Find reopens, at that point all of the previous cycle applications would essentially be "rejected by default"